metadata-ingestion/src/datahub/emitter/rest_emitter.py

```diff
@@ -128,7 +128,14 @@ def emit_mce(self, mce: MetadataChangeEvent) -> None:
 
             response.raise_for_status()
         except HTTPError as e:
-            info = response.json()
+            try:
+                info = response.json()
+            except Exception as parse_exception:
+                response.raise_for_status()
+                raise OperationalError(
+                    "Unable to parse response from ingestion sink",
+                    {"message": str(parse_exception)},
+                ) from e
```

Comment:
Since we're in `except HTTPError as e:`, we know that `response.raise_for_status()` will certainly throw here - probably want to remove that line since we already have it in `e`

We're actually not interested in the parse_exception since it just means that the response body wasn't json. Instead, we should set message to the `response.text` itself